cloudpickle/cloudpickle.py

```diff
@@ -1298,7 +1292,25 @@ def _is_dynamic(module):
         return False
 
     if hasattr(module, '__spec__'):
-        return module.__spec__ is None
+        if module.__spec__ is not None:
+            return False
+
+        # In PyPy, Some built-in modules such as _codecs can have their
+        # __spec__ attribute set to None despite being imported.  For such
+        # modules, the ``_find_spec`` utility of the standard library is used.
+        parent_name = module.__name__.rpartition('.')[0]
+        if parent_name:
+            try:
```

Comment:
codecov complains because this branch is not visited. I could not find any package in the pypy stdlib with `__spec__` set to `None` even when imported (I only found **module** exhibiting that behavior). For the sake of keeping the codebase simple,  maybe we can assume that this wont happen for packages and remove this branch. But no strong opinion whatsoever.
@ogrisel WDYT?

Reply:
LGTM as it is for now.